app/javascript/tests/patient/follow_up_flag/FollowUpFlag.test.js

```diff
@@ -0,0 +1,219 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ReactTooltip from 'react-tooltip';
+import FollowUpFlag from '../../../components/patient/follow_up_flag/FollowUpFlag';
+import ApplyToHousehold from '../../../components/patient/household/actions/ApplyToHousehold';
+import { mockUser1 } from '../../mocks/mockUsers';
+import { mockPatient1, mockPatient3, mockPatient5 } from '../../mocks/mockPatients';
+import { mockJurisdictionPaths } from '../../mocks/mockJurisdiction';
+
+const mockToken = 'testMockTokenString12345';
+const followUpFlagOptions = ['', 'Deceased', 'Duplicate', 'High-Risk', 'Hospitalized', 'In Need of Follow-up', 'Lost to Follow-up', 'Needs Interpretation', 'Quality Assurance', 'Other'];
+
+function getWrapperIndividual(patient, householdMembers) {
+  return shallow(<FollowUpFlag bulk_action={false} current_user={mockUser1} patient={patient} patients={[]} other_household_members={householdMembers} jurisdiction_path="USA, State 1, County 2" jurisdiction_paths={mockJurisdictionPaths} authenticity_token={mockToken} />);
+}
+
+function getWrapperBulkAction(patients) {
+  return shallow(<FollowUpFlag bulk_action={true} current_user={mockUser1} patients={patients} other_household_members={[]} jurisdiction_path="USA, State 1, County 2" jurisdiction_paths={mockJurisdictionPaths} authenticity_token={mockToken} />);
+}
+
+describe('FollowUpFlag', () => {
+  it('Properly renders all main components when not a bulk action', () => {
+    const wrapper = getWrapperIndividual(mockPatient1, []);
+    expect(wrapper.find('.modal-follow-up-flag-body').exists()).toBeTruthy();
```

Comment:
you can just find `Modal.Body` here instead of finding this class which is only added for the tests